--- origsrc/epiphany-2.22.3/configure.ac	2008-06-30 12:58:18.000000000 -0500
+++ src/epiphany-2.22.3/configure.ac	2008-07-10 14:24:16.562500000 -0500
@@ -758,6 +758,22 @@
 # uninstalled share dir to search data
 AC_DEFINE_UNQUOTED([SHARE_UNINSTALLED_DIR], ["`pwd`/data"], [path to source data dir])
 
+# *****
+# Win32
+# *****
+case "$host_os" in
+	cygwin*)
+		platform_win32=yes
+		EPHY_IMPLIB="-lepiphany-$EPIPHANY_API_VERSION"
+		;;
+	*)
+		platform_win32=no
+		EPHY_IMPLIB=
+		;;
+esac
+AC_SUBST(EPHY_IMPLIB)
+AM_CONDITIONAL(PLATFORM_WIN32, test "$platform_win32" = "yes")
+
 # ****************
 # Distributor name
 # ****************
--- origsrc/epiphany-2.22.3/data/epiphany.pc.in	2008-03-28 09:59:45.000000000 -0500
+++ src/epiphany-2.22.3/data/epiphany.pc.in	2008-07-10 14:24:16.578125000 -0500
@@ -14,5 +14,5 @@
 Description: GNOME Web Browser
 Version: @VERSION@
 Requires: gtk+-2.0, libxml-2.0
-Libs: -L${libdir}
+Libs: -L${libdir} @EPHY_IMPLIB@
 Cflags: -I${includedir}
--- origsrc/epiphany-2.22.3/src/Makefile.am	2008-03-28 10:02:20.000000000 -0500
+++ src/epiphany-2.22.3/src/Makefile.am	2008-07-10 14:24:16.656250000 -0500
@@ -209,6 +209,9 @@
 	$(AM_CFLAGS)
 
 epiphany_LDFLAGS = -dlopen self
+if PLATFORM_WIN32
+epiphany_LDFLAGS += -Wl,--export-all-symbols -Wl,--out-implib,libepiphany-$(EPIPHANY_API_VERSION).a
+endif
 
 epiphany_LDADD = \
 	libephymain.la \
@@ -543,3 +546,9 @@
 #check-local: check-python-binding
 
 endif
+
+if PLATFORM_WIN32
+install-data-local:
+	$(mkinstalldirs) $(DESTDIR)$(libdir)
+	$(INSTALL_DATA) libepiphany-$(EPIPHANY_API_VERSION).a $(DESTDIR)$(libdir)
+endif
