--- origsrc/epiphany-2.22.3/doc/reference/Makefile.am	2008-03-28 10:02:20.000000000 -0500
+++ src/epiphany-2.22.3/doc/reference/Makefile.am	2008-07-10 14:24:16.593750000 -0500
@@ -186,6 +186,7 @@
 	$(GECKO_LIBS)	\
 	$(GECKO_EXTRA_LIBS) \
 	$(GECKO_GLUE_LIBS) \
+	$(WEBKIT_LIBS) \
 	$(DBUS_LIBS) \
 	$(LIBINTL)
 
--- origsrc/epiphany-2.22.3/doc/reference/epiphany.types	2007-12-16 07:57:17.000000000 -0600
+++ src/epiphany-2.22.3/doc/reference/epiphany.types	2008-07-10 14:24:16.609375000 -0500
@@ -25,8 +25,6 @@
 #include <ephy-cookie-manager.h>
 #include <ephy-password-manager.h>
 #include <ephy-permission-manager.h>
-#include <mozilla-embed.h>
-#include <mozilla-embed-persist.h>
 
 ephy_embed_get_type
 downloader_view_get_type
@@ -39,8 +37,6 @@
 ephy_cookie_manager_get_type
 ephy_permission_manager_get_type
 ephy_password_manager_get_type
-mozilla_embed_get_type
-mozilla_embed_persist_get_type
 
 #include <ephy-window.h>
 #include <ephy-shell.h>
