--- origsrc/epiphany-2.27.3/src/Makefile.am	2009-09-08 19:41:26.587293900 -0500
+++ src/epiphany-2.27.3/src/Makefile.am	2009-09-08 19:42:17.749325600 -0500
@@ -302,7 +302,7 @@ EPHY_GIR_H_FILES = \
 	$(top_srcdir)/src/ephy-window.h \
 	$(NULL)
 
-Epiphany-$(EPIPHANY_API_VERSION).gir: epiphany $(G_IR_SCANNER)
+Epiphany-$(EPIPHANY_API_VERSION).gir: epiphany$(EXEEXT) $(G_IR_SCANNER)
 	PKG_CONFIG_PATH=$(top_builddir)/data:$$PKG_CONFIG_PATH \
 	$(G_IR_SCANNER) -v \
 		--namespace Epiphany \
@@ -312,7 +312,7 @@ Epiphany-$(EPIPHANY_API_VERSION).gir: ep
 		--include=Gtk-2.0 \
 		--include=libxml2-2.0 \
 		--include=WebKit-1.0 \
-		--program=./epiphany \
+		--program=./epiphany$(EXEEXT) \
 		--output $@ \
 		--pkg epiphany-$(EPIPHANY_API_VERSION) \
 		-DEPIPHANY_COMPILATION \
