--- origsrc/epiphany-2.30.6/src/Makefile.am
+++ src/epiphany-2.30.6/src/Makefile.am
@@ -334,17 +334,19 @@ EPHY_GIR_C_FILES = \
 	$(NULL)
 
 Epiphany-$(EPIPHANY_API_VERSION).gir: $(INTROSPECTION_SCANNER) $(EPHY_GIR_H_FILES) \
-                                      $(EPHY_GIR_C_FILES) epiphany
+                                      $(EPHY_GIR_C_FILES) epiphany$(EXEEXT)
 	$(AM_V_GEN) PKG_CONFIG_PATH=$(top_builddir)/data:$$PKG_CONFIG_PATH \
 	$(INTROSPECTION_SCANNER) -v \
 		--namespace Epiphany \
 		--nsversion=$(EPIPHANY_API_VERSION) \
 		--add-include-path=. \
 		--add-include-path=$(srcdir) \
+		--identifier-prefix=Ephy \
+		--symbol-prefix=ephy_ \
 		--include=Gtk-2.0 \
 		--include=libxml2-2.0 \
 		--include=WebKit-1.0 \
-		--program=./epiphany \
+		--program=./epiphany$(EXEEXT) \
 		--output $@ \
 		--pkg epiphany-$(EPIPHANY_API_VERSION) \
 		-DEPIPHANY_COMPILATION \
