--- origsrc/epiphany-2.30.2/configure.ac	2010-04-08 09:55:44.000000000 -0500
+++ src/epiphany-2.30.2/configure.ac	2010-06-28 03:23:08.182638400 -0500
@@ -366,6 +366,19 @@ AM_CONDITIONAL([ENABLE_NETWORK_MANAGER],
 # uninstalled share dir to search data
 AC_DEFINE_UNQUOTED([SHARE_UNINSTALLED_DIR], ["`pwd`/data"], [path to source data dir])
 
+case "$host_os" in
+	cygwin*)
+		platform_win32=yes
+		EPHY_IMPLIB="-lepiphany-$EPIPHANY_API_VERSION"
+		;;
+	*)
+		platform_win32=no
+		EPHY_IMPLIB=
+		;;
+esac
+AC_SUBST(EPHY_IMPLIB)
+AM_CONDITIONAL(PLATFORM_WIN32, test "$platform_win32" = "yes")
+
 # *******************************
 # Add warning flags
 # *******************************
--- origsrc/epiphany-2.30.2/data/epiphany.pc.in	2010-04-08 09:09:30.000000000 -0500
+++ src/epiphany-2.30.2/data/epiphany.pc.in	2010-06-28 03:23:08.189638800 -0500
@@ -13,5 +13,5 @@ Name: Epiphany Browser
 Description: GNOME Web Browser
 Version: @VERSION@
 Requires: gtk+-2.0, libxml-2.0, webkit-1.0
-Libs: -L${libdir}
+Libs: -L${libdir} @EPHY_IMPLIB@
 Cflags: -I${includedir}
--- origsrc/epiphany-2.30.2/src/Makefile.am	2010-04-08 09:09:41.000000000 -0500
+++ src/epiphany-2.30.2/src/Makefile.am	2010-06-28 03:23:41.821562500 -0500
@@ -157,6 +157,10 @@ epiphany_CFLAGS = \
 
 epiphany_LDFLAGS = -dlopen self
 
+if PLATFORM_WIN32
+epiphany_LDFLAGS += -Wl,--out-implib,libepiphany-$(EPIPHANY_API_VERSION).a
+endif
+
 epiphany_LDADD = \
 	libephymain.la \
 	$(top_builddir)/src/bookmarks/libephybookmarks.la \
@@ -368,3 +372,9 @@ Epiphany-$(EPIPHANY_API_VERSION).typelib
 CLEANFILES += $(gir_DATA) $(typelibs_DATA)
 
 endif # HAVE_INTROSPECTION
+
+if PLATFORM_WIN32
+install-data-local:
+	$(mkinstalldirs) $(DESTDIR)$(libdir)
+	$(INSTALL_DATA) libepiphany-$(EPIPHANY_API_VERSION).a $(DESTDIR)$(libdir)
+endif
