--- origsrc/epiphany-3.4.1/src/Makefile.am	2012-05-14 20:46:42.532657600 -0500
+++ src/epiphany-3.4.1/src/Makefile.am	2012-05-14 20:47:57.375938400 -0500
@@ -278,7 +278,7 @@ EPHY_GIR_C_FILES = \
 	$(NULL)
 
 Epiphany-$(EPIPHANY_API_VERSION).gir: $(INTROSPECTION_SCANNER) $(EPHY_GIR_H_FILES) \
-                                      $(EPHY_GIR_C_FILES) epiphany
+                                      $(EPHY_GIR_C_FILES) epiphany$(EXEEXT)
 	$(AM_V_GEN) PKG_CONFIG_PATH=$(top_builddir)/data:$$PKG_CONFIG_PATH \
 	$(INTROSPECTION_SCANNER) -v --warn-all \
 		--identifier-prefix=Ephy \
@@ -290,7 +290,7 @@ Epiphany-$(EPIPHANY_API_VERSION).gir: $(
 		--include=Gtk-3.0 \
 		--include=libxml2-2.0 \
 		--include=WebKit-3.0 \
-		--program=./epiphany \
+		--program=./epiphany$(EXEEXT) \
 		--output=$@ \
 		--pkg=epiphany-$(EPIPHANY_API_VERSION) \
 		-DEPIPHANY_COMPILATION \
